pyproject.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
     "python-dotenv>=1.0.0",
     "appdirs>=1.4.4",
     "jsonref>=1.1.0",
+    "mem0ai>=0.1.24",
```

Comment:
This should be an optional dependency that users who want to use mem0 can add.

Reply:
Yes, moved to optional dependency.